lib/core/src/Cardano/Wallet.hs

```diff
@@ -1388,15 +1393,16 @@ selectAssetsNoOutputs
     -> ExceptT ErrSelectAssets IO result
 selectAssetsNoOutputs ctx wid wal tx transform = do
     -- NOTE:
-    -- Could be made nicer by allowing 'performSelection' to run with no target
+    -- Could be made nicer by allowing 'runWalletCoinSelection' to run with no target
     -- outputs, but to satisfy a minimum Ada target.
     --
     -- To work-around this immediately, I am simply creating a dummy output of
     -- exactly the required deposit amount, only to discard it on the final
     -- result. The resulting selection will therefore have a delta that is at
     -- least the size of the deposit (in practice, slightly bigger because this
     -- extra outputs also increases the apparent minimum fee).
-    deposit <- calcMinimumDeposit @_ @s @k ctx wid
+    deposit <- withExceptT ErrSelectAssetsNoSuchWallet $
+        calcMinimumDeposit @_ @s @k ctx wid
```

Comment:
Hi @rvl, are you planning to move the `dummyOutput` logic into the `CoinSelection` module as part of this PR?